src/components/Setting/OrcaCoinConnection.tsx

```diff
@@ -0,0 +1,3 @@
+export default function OrcaCoinConnection() {
+  return <div className="orca-coin-connection">OrcaCoin Connected</div>;
```

Comment:
Until we come up with a design for the OrcaCoin Connection, we can leave it like this.